mail/api.py

```diff
@@ -127,19 +112,27 @@ def send_batch(cls, subject, body, recipients,  # pylint: disable=too-many-argum
                along with recipients we failed to send to.
         """
         original_recipients = recipients
-        body, recipients = cls._recipient_override(body, original_recipients)
+
+        if settings.MAILGUN_RECIPIENT_OVERRIDE is not None:
```

Comment:
I'm not sure it's necessary, and maybe there's the possibility that we want to set this environment variable on a PR build where `DEBUG` is false